packages/create-remix/rollup.config.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const path = require("path");
+const babel = require("@rollup/plugin-babel").default;
+const nodeResolve = require("@rollup/plugin-node-resolve").default;
+const copy = require("rollup-plugin-copy");
+
+const {
+  buildDir,
+  copyToPlaygrounds,
+  createBanner,
+  executableBanner,
+  getVersion,
+} = require("../../rollup.utils");
+
+/** @returns {import("rollup").RollupOptions[]} */
+module.exports = function rollup() {
+  let sourceDir = path.relative(process.cwd(), __dirname) || ".";
+  let outputDir = path.join(buildDir, "node_modules/create-remix");
```

Comment:
Using `buildDir` from `rollup.utils` ensure's we respect `LOCAL_BUILD_DIRECTORY`